tests/phpunit/tests/Shortcode/Stories_Shortcode.php

```diff
@@ -131,10 +131,11 @@ public function test_prepare_story_attrs() {
 			'show_date'                 => true,
 			'show_excerpt'              => true,
 			'show_story_archive_link'   => false,
-			'show_story_archive_label'  => true,
+			'stories_archive_label'     => 'Stories',
```

Comment:
```suggestion
			'stories_archive_label'     => __( 'Stories', 'web-stories' ),
```

Reply:
This is a test file, no need for translation here.

Reply:
🤦